.github/workflows/announce-release.yml

```
@@ -12,7 +12,8 @@ permissions:
 
 jobs:
   post_to_mastodon:
-    if: contains(github.event.pull_request.title, 'chore(main): release')
+    if: |
```

actionlint (1.7.4)

if: condition "${{ contains(github.event.head_commit.message, 'chore(main): release') }}\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)